templates/zerver/emails/onboarding_team_to_zulip.txt

```diff
@@ -0,0 +1,18 @@
+{% trans %}If you've already decided to use Zulip for your organization, welcome! You can use our guide for setting up your organization to get started.{% endtrans %} [ Guide to setting up your organization: {{ get_organization_started }} ]
```

Comment:
Could we cut the text labels for the URLs? I feel like in plain-text format, our generally fairly nice URLs speak for themselves as to what it'll do, and it feels somewhat redundant.